plugins/woocommerce/tests/e2e-pw/tests/merchant/create-coupon.spec.js

```diff
@@ -39,102 +39,110 @@ const test = baseTest.extend( {
 	},
 } );
 
-test.describe( 'Coupon management', { tag: '@services' }, () => {
-	for ( const couponType of Object.keys( couponData ) ) {
-		test( `can create new ${ couponType } coupon`, async ( {
-			page,
-			coupon,
-		} ) => {
-			await test.step( 'add new coupon', async () => {
-				await page.goto(
-					'wp-admin/post-new.php?post_type=shop_coupon'
-				);
-				await page
-					.getByLabel( 'Coupon code' )
-					.fill( couponData[ couponType ].code );
-				await page
-					.getByPlaceholder( 'Description (optional)' )
-					.fill( couponData[ couponType ].description );
-				await page
-					.getByPlaceholder( '0' )
-					.fill( couponData[ couponType ].amount );
-
-				// set expiry date if it was provided
-				if ( couponData[ couponType ].expiryDate ) {
+test.describe(
+	'Coupon management',
+	{ tag: [ '@services', '@skip-on-default-wpcom' ] },
```

Comment:
Deleting coupons returns 500 for all coupon tests on WPCOM. There was [a PR](https://github.com/woocommerce/woocommerce/pull/47474) to fix that, but it's still happening for this env. For now, I'm skipping these tests.